### DISCUSSION TITLE
Any plans for Wildpromts?

### DISCUSSION THREAD

**Discussion by DavidKouw** [lllyasviel/Fooocus#2737] [Q&A] [answered]
Hi there! thanks for work with Foo. #2102 Here no news about any progress from author. So I wanna ask if its possible to be Wildprompts implemented in Main Foo?  Its rly cool features in pair with wildcards, and I using it in oldversion fork, But it's really lacking in the main app. I trivially lack the knowledge of how to put this into the main foooo without breaking anything xD

**Comment by mashb1t** [comment, marked as answer]
Please check https://github.com/lllyasviel/Fooocus/discussions/2739#discussioncomment-9075824 for a little update / statement.
Currently it's not on the roadmap.
